cmd/config-current.go

```diff
@@ -212,6 +214,7 @@ func newServerConfig() *serverConfig {
 	srvCfg.Cache.Drives = make([]string, 0)
 	srvCfg.Cache.Exclude = make([]string, 0)
 	srvCfg.Cache.Expiry = globalCacheExpiry
+	srvCfg.Cache.MaxUse = globalCacheMaxUse
```

Comment:
This requires a change in config version @annanay25 PTAL at config-migrate.go, config-versions.go for more details on how to achieve that. Feel free to join us on https://slack.minio.io if you further questions.